pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/Study.java

```diff
@@ -1,48 +1,26 @@
 package org.broadinstitute.dsm.model;
 
-import java.util.Arrays;
-import java.util.List;
-import java.util.NoSuchElementException;
-
-import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.exception.DsmInternalError;
 
 public enum Study {
     ATCP("ATCP"),
     RGP("RGP"),
     SINGULAR("SINGULAR"),
     LMS("LMS"),
```

Comment:
Actually, I believe Sampath is correct, since we are using Study (in this case) to validate study GUIDs provided in messages from DSS. In that case, I believe cmi-lms is correct (or CMI-LMS in the case of this class). 
The reason I touched this class at all was because a couple things were wrong in the code, and then noticed there was some incorrect data, and I removed that. The change Sampath suggests fits with that theme.

We have a couple options: fix or remove "LMS", or more drastically remove the class and use something else in its place. The downside of the class is that it is not tied to a DB table, so if/when changes are made to ddp_instance they are not reflected in Study unless someone remembers to do that (and it appears that's what led to this problem).
The class is lightly used (also in AddFamilyMember and AddFamilyMemberRoute), but perhaps the best thing to do at this point in Pepper's lifecycle is to just fix Study for now. 
What do you think?

Reply:
I agree! I'm sorry if I confused you before, the thing is, when we publish to dsm-tasks to run the export to ES, we call by instance name, maybe can you put a comment in the code so we can know which calls are by guid and which calls are by name? It will make future handling of the code or cases of manual publishing to the topic much easier

Reply:
Let me see if I understand, since I *am* concerned about this code and approach, and appreciate your insights. The use of Study in DSMtasksSubscription is to correctly identify the study referred to in message attributes map. In the particular case addressed in this PR, that message typically comes from DSS, and should match ddp_instance.study_guid (of course, that is just by convention, since we do not share a common source of truth for that, as far as I know).
Are you saying there are other instances where we send a message processed by DSMtasksSubscription that may use a different convention for identifying studies (that is, by instance name)?

Reply:
The more I look at this, the messier it is...

After a bit more investigation, how we identify a ddp_instance based on message contents varies by message task type (in DSMtasksSubscription), which is unfortunate. Ideally we would have one convention. Even better, we fix having two different non-key identifiers for a ddp_instance (that is, either instance name or study guid, not both). 

Getting that straightened out is a fairly large task that includes DSM, DSS, and the conventions used when sending messages by hand. We also need to see where we depend on a particular value of instance name or study guid (the Study class is just one example). And we also have to consider where we display instance names to users (rather than the display_name).

For the purposes of this PR, I feel like my instincts were correct in trying to delete the unused portions of Study. If this class has a future where it is used more broadly, it should perhaps be moved to ddp_common, so at least DSM and DSS are using the same source. I will add a comment to Study saying that we should not expand its use. I will also attempt to comment any of the message handlers that seem unclear on whether they are handling a study GUID or instance name, but it seems those are pretty clear -- just painfully inconsistent.

However, there is an inconsistency in BasicDefaultDataMaker that I will fix for this PR.